.github/workflows/ci.yml

```diff
@@ -0,0 +1,111 @@
+# TODO(#158): to be replaced with real service integration test.
+name: 'ci'
```

Comment:
What's the difference between this one and the old `test-integration.yml` and `test-unit.yaml`?

Reply:
Just following the same pattern we did for lumberjack and pmap.